### PR TITLE
Update link to our Privacy Policy history in our Privacy Policy

### PR DESCRIPTION
**Pull request by Ndpnt**

### CODE REVIEW

**Review by clementbiron — APPROVED** [review]

File: content/pages/en/privacy-policy.mdx [hunk 1]
```diff
@@ -5,7 +5,7 @@ html_description: This privacy policy explains how, why and under what condition
 
 <p className="text__smallcaps mb__3XL">Last updated: September 26, 2023</p>
```
Comment (MattiSG) [inline content/pages/en/privacy-policy.mdx:6]: So, should we update the “last updated” date for such a minor change or not? 😉

File: content/pages/en/privacy-policy.mdx [hunk 2]
```diff
@@ -5,7 +5,7 @@ html_description: This privacy policy explains how, why and under what condition
...
 
-This privacy policy explains how, why and under what conditions the Open Terms Archive (OTA) site collects personal information and how it is used. Our privacy policy will change over time. And of course, we also record the changes of [our own documents](https://github.com/OpenTermsArchive/demo-versions/tree/main/Open%20Terms%20Archive) 😉
+This privacy policy explains how, why and under what conditions the Open Terms Archive (OTA) site collects personal information and how it is used. Our privacy policy will change over time. And of course, we also record the changes of [our own privacy policy](https://github.com/OpenTermsArchive/demo-versions/commits/main/Open%20Terms%20Archive/Privacy%20Policy.md) 😉
```
Comment (MattiSG) [inline content/pages/en/privacy-policy.mdx:8]:
```suggestion
This privacy policy explains how, why and under what conditions the Open Terms Archive (OTA) site collects personal information and how it is used. Our privacy policy will change over time. And of course, we [publicly record those changes](https://github.com/OpenTermsArchive/demo-versions/commits/main/Open%20Terms%20Archive/Privacy%20Policy.md) with Open Terms Archive 😉
```

File: content/pages/fr/privacy-policy.mdx [hunk 3]
```diff
@@ -6,7 +6,7 @@ permalink: "/politique-de-confidentialite"
 
 <p className="text__smallcaps mb__3XL">Dernière mise à jour : 26 septembre 2023</p>
 
-Cette politique de confidentialité explique de quelle manière, pour quelles raisons et dans quelles conditions le site d'Open Terms Archive (OTA) collecte des informations personnelles, et la finalité du traitement de ces dernières. Notre politique de confidentialité est vouée à évoluer au fil du temps. Et bien sûr, nous enregistrons également les changements de [nos propres documents](https://github.com/OpenTermsArchive/demo-versions/tree/main/Open%20Terms%20Archive) 😉
+Cette politique de confidentialité explique de quelle manière, pour quelles raisons et dans quelles conditions le site d'Open Terms Archive (OTA) collecte des informations personnelles, et la finalité du traitement de ces dernières. Notre politique de confidentialité est vouée à évoluer au fil du temps. Et bien sûr, nous enregistrons également les changements de [notre propre politique de confidentialité](https://github.com/OpenTermsArchive/demo-versions/commits/main/Open%20Terms%20Archive/Privacy%20Policy.md) 😉
```
Comment (MattiSG) [inline content/pages/fr/privacy-policy.mdx:9]:
```suggestion
Cette politique de confidentialité explique de quelle manière, pour quelles raisons et dans quelles conditions le site d'Open Terms Archive (OTA) collecte des informations personnelles, et la finalité du traitement de ces dernières. Notre politique de confidentialité est vouée à évoluer au fil du temps. Et bien sûr, nous [enregistrons publiquement ces changements](https://github.com/OpenTermsArchive/demo-versions/commits/main/Open%20Terms%20Archive/Privacy%20Policy.md) avec Open Terms Archive 😉
```

**Review by MattiSG — COMMENTED** [review]
I find the original suggestion very redundant (three times “privacy policy” in three sentences in a row).

File: content/pages/en/privacy-policy.mdx (hunk 1 above)
Comment (Ndpnt) [reply content/pages/en/privacy-policy.mdx:6]: Yep we should

Comment (Ndpnt) [reply content/pages/en/privacy-policy.mdx:6]: "Last update" date is not related to the importance of the changes for me.

**Pull request merged**